.NET/Microsoft.Recognizers.Text.Number/Swedish/Extractors/NumberExtractor.cs

```diff
@@ -6,28 +6,22 @@
 
 namespace Microsoft.Recognizers.Text.Number.Swedish
 {
-    public class NumberExtractor : BaseNumberExtractor
+    public class NumberExtractor : CachedNumberExtractor
     {
         private const RegexOptions RegexFlags = RegexOptions.Singleline | RegexOptions.ExplicitCapture;
 
         private static readonly ConcurrentDictionary<(NumberMode, NumberOptions), NumberExtractor> Instances =
             new ConcurrentDictionary<(NumberMode, NumberOptions), NumberExtractor>();
 
-        private readonly NumberMode mode;
+        private readonly string keyPrefix;
 
         private NumberExtractor(BaseNumberOptionsConfiguration config)
             : base(config.Options)
         {
 
-            this.mode = config.Mode;
+            keyPrefix = string.Intern(ExtractType + "_" + config.Options + "_" + config.Mode + "_" + config.Culture);
 
-            NegativeNumberTermsRegex = new Regex(NumbersDefinitions.NegativeNumberTermsRegex + "$", RegexFlags);
-
-            AmbiguousFractionConnectorsRegex = new Regex(NumbersDefinitions.AmbiguousFractionConnectorsRegex, RegexFlags);
```

Comment:
Are these two removed as no longer relevant? Or not implemented in Swedish?
I need to do a comparison with English before completing the review.

Reply:
So far they've not been used. But, I guess you have a point in letting them remain. I'm working w/ the Range implementation and also improving the fraction recognizer now and can it be that they could potentially need these? I haven't checked yet. Once again - it might be better keeping them for the future.

Reply:
Let's bring it back here then. Thanks.